plugin/plugin.go

```diff
@@ -105,5 +105,8 @@ const Namespace = "coredns"
 // TimeBuckets is based on Prometheus client_golang prometheus.DefBuckets
 var TimeBuckets = prometheus.ExponentialBuckets(0.00025, 2, 16) // from 0.25ms to 8 seconds
 
+// SlimTimeBuckets is low cardinality set of duration buckets.
+var SlimTimeBuckets = prometheus.ExponentialBuckets(0.00025, 10, 5) // from 0.25ms to 2.5 seconds
```

Comment:
I seen this be 6s in k8s deployments.....

Reply:
Are you talking about the health endpoint? That just means it fits in the `+Inf` bucket.

But, how can it be 6s when the timeout is 3s?